python/test/t_ContinuousPC_std.py

```diff
@@ -29,8 +29,8 @@ def testSpecificInstance():
     # print(skel.toDot())
 
     dag = learner.learnDAG()
-    print(dag.toDot(), flush=True)
-
+    print(dag.toDot())
+    sys.stdout.flush()
```

Comment:
```suggestion
print(dag.toDot(),flush=True)
```

Reply:
I tried it, unfortunately that wouldnt work on py2k

Reply:
oh ... 
```python
from __future__ import print_function
``` 
??
(in order to have py3 (rather than py2)'s code)

Reply:
the flush argument is not available, it was introduced much later, in python 3.4